mod_app/models/film_model.py

```diff
@@ -31,18 +30,28 @@ def __str__(self):
         related_name="source_material_link",
     )
     genre = models.ManyToManyField(Tag, related_name="genres", blank=True)
+
     bfi_category = models.CharField(
         max_length=100, blank=True, null=True
     )  # can use choices if preset
 
-    actors = models.ManyToManyField(
-        "Actor", related_name="films", blank=True, verbose_name="Cast"
+    cast = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+    crew = models.CharField(
+        max_length=255, blank=True, null=True, verbose_name="Credits"
```

Comment:
I'm not sure removing the actor model entirely is called for - you could have kept a simpler model with just the name. That'd leave the door open for searching/filtering by people later. How about:

```python
class Roles(models.TextChoices):
    ACTOR = "AC", "Actor"
    DIRECTOR = "DI", "Director"
    SCREENPLAY = "SP", "Screenplay"
    SCENOGRAPHY = "SG", "Scenography"
    PHOTOGRAPHY = "PH", "PHOTOGRAPHY"


class Person(models.Model):
    name = models.CharField(max_length=255, blank=False)
    role = models.CharField(max_length=2, blank=False, choices=Roles.choices)
```

Then on Film:
```python
class Film(models.Model):
    cast = models.ManyToManyField(
        "Person", related_name="actor_in", blank=True, verbose_name="Cast", limit_choices_to={"role": Roles.ACTOR})
    credits = models.ManyToManyField(
        "Person", related_name="credited_in", blank=True, verbose_name="Crew")
```

That'd give you the flexibility to create a person page later which lists all their various roles.

Reply:
ooh yes doing it by person which can be cast/credits might be better. I'm not sure if we should have all the roles laid out since those might be variable but maybe actor and director?

Reply:
Hmm, actually if people can have different roles on different films, that property needs to be on a through model. Then you'd need tabular inline to show it in the form, which means it gets displayed at the end of the page after the fieldsets layout. Could maybe have general cast and credits section? Or ignore me and go with what's currently working.

Reply:
I think for now I'll just leave it as strings, maybe a textfield but it feels like we'd be over complicating the need for models there since it would only be quite basic information and they don't seem interested in grouping or searching by that (currently)